lib/board.rb

```diff
@@ -0,0 +1,76 @@
+require 'pry'
+class Board
+  attr_accessor :cells
+
+  def initialize
+    @cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+    reset!
+  end
+
+  def reset!
+    cells.clear
+    cells << [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+    cells.flatten!
+  end
+
+  def display
+    puts " #{cells[0]} | #{cells[1]} | #{cells[2]}  "
+    puts "-----------"
+    puts " #{cells[3]} | #{cells[4]} | #{cells[5]}  "
+    puts "-----------"
+    puts " #{cells[6]} | #{cells[7]} | #{cells[8]}  "
+  end
+
+  def full?
+    cells.all? do |cell|
```

Comment:
No need to use an if statement to return a boolean, the same could be accomplished with:
```
cells.all?{|token| token == "X" || token == "O"}
```